sentinelhub/decoding.py

```diff
@@ -31,7 +31,10 @@ def decode_data(response_content, data_type):
     :raises: ValueError
     """
     if data_type is MimeType.JSON:
-        return json.loads(response_content.decode('utf-8'))
+        response_text = response_content.decode('utf-8')
```

Comment:
If the returned content is non-empty and cannot be decoded to JSON then the line `return json.loads(response_text)` returns an error and I think that is ok.

The only case that I wanted to cover with this change is that when a response is empty this still gets treated as an ok JSON format and decoded into an empty string. Otherwise `json.loads('')` would return an error.

Reply:
Ah, true.